packages/react-data-grid/src/masks/InteractionMasks.tsx

```diff
@@ -594,11 +587,29 @@ export default class InteractionMasks<R, K extends keyof R> extends React.Compon
   };
 
   onDragHandleDoubleClick = (): void => {
-    this.props.onDragHandleDoubleClick(this.state.selectedPosition);
+    const { selectedPosition } = this.state;
+    const { columns, onGridRowsUpdated, rowGetter, rowsCount } = this.props;
+    const column = columns[selectedPosition.idx];
+    const value = getSelectedCellValue({ selectedPosition, columns, rowGetter });
+    const cellKey = column.key;
+
+    onGridRowsUpdated({
+      cellKey,
+      fromRow: selectedPosition.rowIdx,
+      toRow: rowsCount - 1,
+      updated: { [cellKey]: value } as never,
+      action: UpdateActions.COLUMN_FILL
+    });
   };
 
-  onCommit = (args: CommitEvent<R>): void => {
-    this.props.onCommit(args);
```

Comment:
`onCommit` and `onDragHandleDoubleClick`  call `onGridRowsUpdated` so I just moved the logic here